notebooks/04_model_select_and_optimize/nni_ncf.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>it came as a surprise that you are comparing both algos instead of doing hyperparameter tuning on NCF. I think for simplicity it would be better to have a notebook with hyperparameter tuning for NCF. </p><p> </p><p>About having a comparison between SVD and NCF, is there a benefit of having 1 notebook with the comparison vs comparing the outputs of both independent notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Originally I was thinking that this new notebook would focus more on the comparison between model architectures after they've been tuned through NNI. With only the SVD and NCF model though comparing the outputs of the two notebooks may serve the same purpose. I'll refactor the notebook to focus on tuning for NCF, thanks for the suggestion!</p>

Reply:
could you please ping me here when the change is done?